styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,87 @@
 .footer{
-  
-}
\ No newline at end of file
+  ul{
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+  .footer-menu {
+    li {
+      list-style: none;
+      padding: 0;
+      margin: 8px 0;
+      a {
```

Comment:
I am always in favor of styling classes over elements. If we are good with Drupal then I say take the plunge.